.devcontainer/devcontainer.json

```diff
@@ -3,14 +3,18 @@
 		"CoenraadS.bracket-pair-colorizer-2",
 		"xaver.clang-format",
 		"notskm.clang-tidy",
-		"ms-vscode.cpptools",
 		"eamodio.gitlens",
 		"yzhang.markdown-all-in-one",
 		"christian-kohler.path-intellisense",
 		"vscodevim.vim",
 		"vscode-icons-team.vscode-icons",
 		"ms-vsliveshare.vsliveshare",
-		"ms-azuretools.vscode-docker"
+		"ms-azuretools.vscode-docker",
+		"llvm-vs-code-extensions.vscode-clangd",
+        "mitaki28.vscode-clang",
```

Comment:
nit: whitspace alignment changed? Don't have an opinion what whitespace is nominal here, but lets matchy matchy one way or the other (maybe you have to fix up my own whitespace and didn't want the PR polluted by diff lines?).